src/VirtoCommerce.CustomerExportImportModule.Data/Validation/ImportAccountValidator.cs

```diff
@@ -16,12 +17,14 @@ public class ImportAccountValidator : AbstractValidator<ImportRecord<ImportableC
         private readonly UserManager<ApplicationUser> _userManager;
         private readonly IStoreSearchService _storeSearchService;
         private readonly ISettingsManager _settingsManager;
+        private readonly ImportRecord<ImportableContact>[] _allRecords;
 
-        public ImportAccountValidator(UserManager<ApplicationUser> userManager, IStoreSearchService storeSearchService, ISettingsManager settingsManager)
+        public ImportAccountValidator(UserManager<ApplicationUser> userManager, IStoreSearchService storeSearchService, ISettingsManager settingsManager, ImportRecord<ImportableContact>[] allRecords)
         {
             _userManager = userManager;
             _storeSearchService = storeSearchService;
             _settingsManager = settingsManager;
+            _allRecords = allRecords;
```

Comment:
_allContacts?

Reply:
This is not domain models, so we use record term everywhere in this case to prevent confusion

Reply:
(and they have type `ImportRecord<ImportableContact>`)